roles/tas_single_node/templates/manifests/fulcio/fulcio-server.yaml

```diff
@@ -33,12 +33,12 @@ spec:
             - --fileca-cert
             - /var/run/fulcio-secrets/cert.pem
             - --fileca-key-passwd
-            - {{ tas_single_node_fulcio_ca_passphrase  }}
-            - --ct-log-url=http://ctlog-pod:6962/{{ tas_single_node_ct_logprefix }}
+            - "{{ tas_single_node_fulcio_ca_passphrase }}"
+            - --ct-log-url=http://ctlog-pod:6962/"{{ tas_single_node_ct_logprefix }}"
```

Comment:
I think it's better to put the quotes around the whole string, not just part of it - I don't think I've ever seen usage of quotes mid-string and I'd be worried this is fragile and could behave differently on different versions of Ansible/Jinja2

Reply:
Change made